datadog_cluster_agent/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
We should document `datadog.cluster_agent.prometheus.health`, see example
https://github.com/DataDog/integrations-core/blob/master/cilium/assets/service_checks.json